api/inc/vmpu_exports.h

```diff
@@ -155,6 +155,24 @@ typedef struct {
     uint32_t acl_count;
 } UVISOR_PACKED UvisorBoxConfig;
 
+typedef struct
+{
+    /* Pointer to the user context */
+    void * ctx;
+    /* Pointer to the box heap */
```

Comment:
@niklas-arm What do you think of the mixed use of box/process semantics? Shall we use only "box" or is there another reason behind the `process_` names? (Besides the original idea of having processes in CMSIS).